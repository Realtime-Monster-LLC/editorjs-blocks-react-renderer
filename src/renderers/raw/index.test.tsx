import React from 'react';
import { create } from 'react-test-renderer';
import Raw, { RawBlockData } from '.';

describe('<Raw />', () => {
  describe('when receives a raw block', () => {
    const data: RawBlockData = {
      html: '<div style="background: #000; color: #fff; font-size: 30px; padding: 50px;">Any HTML code</div>',
    };

    it('renders content as HTML', () => {
      expect(create(<Raw data={data} />).toJSON()).toMatchSnapshot();
    });
  });
});
