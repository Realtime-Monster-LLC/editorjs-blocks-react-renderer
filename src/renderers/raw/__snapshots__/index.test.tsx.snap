// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Raw /> when receives a raw block renders content as HTML 1`] = `
<div
  style={
    Object {
      "background": "#000",
      "color": "#fff",
      "fontSize": "30px",
      "padding": "50px",
    }
  }
>
  Any HTML code
</div>
`;
